gnocchi/rest/__init__.py

```diff
@@ -665,6 +667,8 @@ def post(self):
         except indexer.NoSuchResource as e:
             abort(404, e)
 
+        return r.metrics
```

Comment:
i think this will return all metrics in resource not just the new metrics you added. it's failing in gate in gate because the new metric is not necessarily the first item returned.

if you're testing this locally, you can add verbose: true to gabbi tests, so check what order metrics are returned in (i'm guessing alphabetical). something like:

     - name: create new metrics
       verbose: True
       POST: /v1/resource/generic/f93450f2-d8a5-4d67-9985-02511241e7d1/metric
       request_headers:
           x-user-id: 0fbb231484614b1a80131fc22f6afc9c
           x-project-id: f3d41b770cc14f0bb94a1d5be9c0e3ea
           content-type: application/json
        data:
          foobar:
            archive_policy_name: low
       status: 200
       response_json_paths:
           $[0].name: foobar
           $[0].resource_id: f93450f2-d8a5-4d67-9985-02511241e7d1